gradle.properties

```diff
@@ -1,7 +1,7 @@
 org.gradle.caching=true
 group=org.ballerinalang
 version=1.0.3-SNAPSHOT
-ballerinaLangVersion=2.0.0-Preview6
+ballerinaLangVersion=2.0.0-Preview7-SNAPSHOT
 ballerinaTomlParserVersion=1.2.2
 stdlibSystemVersion=0.6.2
```

Comment:
We have to update the dependent module versions too.

Reply:
Updated in 5a7b22f